docker_run.bash

```diff
@@ -17,7 +17,7 @@ usage() {
 [ $# -eq 0 ] && usage
 
 alerter="email"
-default_image="ericjmarti/inventory-hunter:latest"
+default_image="phikai/inventory-hunter:latest"
```

Comment:
Hmm... this shouldn't be here, it was only for my fork, but it was part of this branch.

```suggestion
default_image="ericjmarti/inventory-hunter:latest"
```